docker-compose/docker-compose.yaml

```diff
@@ -550,6 +550,26 @@ services:
     restart: always
     stop_grace_period: 120s
 
+  # Description: This container will collect and expose prometheus metrics about `pgsql` PostgreSQL server.
+  #
+  # Disk: None
+  # Ports exposed to other Sourcegraph services: 9187/TCP
+  # Ports exposed to the public internet: none
+  #
+  # If you're using a DB instance hosted outside of docker-compose, the environment variables
+  # for this container will need to be updated to reflect the new connection information.
+  pgsql-exporter:
+    container_name: pgsql-exporter
+    image: 'index.docker.io/sourcegraph/postgres_exporter:3.37.0@sha256:20e58b62f064037ac3d901eba565f49d7e1daae2a237e6fa3d5351580d576dea'
+    cpus: 0.1
+    mem_limit: '50m'
```

Comment:
I don't think exporter is really resource extensive

https://github.com/sourcegraph/deploy-sourcegraph/blob/f629296cddc1ec84cb31b95e7c874160d05b6bc5/base/pgsql/pgsql.Deployment.yaml#L83-L89

Reply:
let's maybe give it 512m and 0.5 cpu?

Reply:
I take it back, our base deployment has these values: https://github.com/sourcegraph/deploy-sourcegraph/blob/master/base/codeintel-db/codeintel-db.Deployment.yaml#L83

Reply:
We can always change it if we find any problem with prom scraping stuff from the exporter (we actually have a graph about scraping errors on the dashboard)